modules/c3-assisted-installer-preparing-bastion-server.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_oci/installing-c3-assisted-installer.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="c3-ai-preparing-bastian-server_{context}"]
+= Preparing the {oci} Bastion server
+
+By implementing a bastion host, you can securely and efficiently manage access to your {oci-c3-no-rt} resources, ensuring that your private instances remain protected and accessible only through a secure, controlled entry point.
+
+.Prerequisites
+
+* See link:https://www.oracle.com/a/otn/docs/compute_cloud_at_customer_agent_based_installation.pdf?source=:em:nl:mt::::PCATP[Bastion server - prerequisites].
```

Comment:
```suggestion
* See the Prerequisites section in the  link:https://www.oracle.com/a/otn/docs/compute_cloud_at_customer_agent_based_installation.pdf?source=:em:nl:mt::::PCATP[Oracle documentation].
```
Because we don't have a way to provide a direct link to each section in the Oracle guide, I revised the link text here and in other places so that customers won't expect, for example, to click the link and immediately see "Prerequisites".